db/queryUserData.ts

```diff
@@ -0,0 +1,31 @@
+import { IArgumentsRetrieve, IUserDetails } from '../@types/types'
+import { db } from './getDbInstance'
+
```

Comment:
I'm pretty sure this is not respecting the 100 chars limit.
We prefer a more vertical style, for example:
```
argv.language
? (...)
: (...);
```

Also, what's with the `1=1` ? 

Also, why are you using raw (unescaped) values? What is the risk you take by doing that?

Reply:
Actually there are other rules called ignoreTemplateLiterals and ignoreStrings that make it possible to use more than the limit chars when using template literals.

The 1=1 is a way to always get a true condition when not receiving one of those parameters. It's commonly used in T-SQL i think.

I changed raw values as they are really risk for SQL Injection.